.github/ISSUE_TEMPLATE/default_issue.yml

```diff
@@ -1,7 +1,6 @@
 name: Default Issue
 description: File an issue for a new feature or bug
 title: "[Issue]: "
-projects: "Grants Team Agile Planning"
```

Comment:
This was noted in https://github.com/usdigitalresponse/grants-ingest/pull/14#discussion_r1145123514 but didn't get implemented in that PR due to auto-merge.